src/components/flat-table/sort/sort.style.ts

```diff
@@ -45,4 +40,13 @@ const StyledSpaceHolder = styled.div`
   width: 22px;
 `;
 
-export { StyledSort, StyledSpaceHolder };
+interface StylesSortIconProps extends StyledIconProps {
+  iconColor?: string;
+}
+
+const StyledSortIcon = styled(Icon)<StylesSortIconProps>`
+  padding-left: 6px;
```

Comment:
**suggestion(non-blocking):** feel free to leave as is or if you want to use a design token here, the value for 6px is `spacing075`. I'm happy either way though.